bluemira/__init__.py

```diff
@@ -23,6 +23,16 @@
 Initialise the bluemira package.
 """
 
+import os
+import sys
+
 from . import _version  # noqa: ABS101
 
+# this a change
+if os.environ.get("BLUEMIRA_ENV", "") == "CONTAINER":
+    sys.path.append("/usr/lib/freecad/Ext")
+    sys.path.append("/usr/lib/freecad/Mod/Part")
+    sys.path.append("/usr/lib/freecad/Mod/Draft")
+    sys.path.append("/usr/lib/python3/dist-packages")
```

Comment:
This last line feels bad for when we generalise this. Is it possible to only install python packages in the user venv or is something we need installed here?

Reply:
It is pretty terrible and I meant to leave a comment on top the line.

But it's not as bad as it seems because ur appending it last, any packages on the path prior (say numpy) will come from the vevn you're using if they exist on in both i.e. because it's last it has lowest import priority.

Apt-install'ing Freecad puts the packages it needs (PySide and friends) in the global dist-packages. I've tried to make it put them in active venv but it won't work.

I thought it might be possible to copy the packages from the global dist-packages to the venv but I thought it probably wouldn't work because of paths and maybe other files, but I could be wrong.
I didn't try and just did this instead.

I think this is fine for a container but another solution will be needed if we want bluemira to be packageable and link against an installed Freecad.
It might be possible to include only the needed packages in the path (i.e. append PySide etc.) from the global, but also haven't tried.

Reply:
Fair enough! For now a comment will be enough for me. 

It would be nice in future if we only need to add specific packages to the system path, sounds like the best option given other constraints (if its possible).